access/ms-teams/validate.go

```diff
@@ -134,3 +117,28 @@ func validate(configPath, userID string) error {
 
 	return nil
 }
+
+func loadConfigAndCheckApp(configPath string) (*Bot, *Config, error) {
```

Comment:
:nail_care: 
I would probably separate this in two methods
`LoadConfig` and `CheckApp`

Reply:
Done